.harness/pythonsdkpublish.yaml

```diff
@@ -0,0 +1,89 @@
+inputSet:
+  name: python-sdk-publish
+  identifier: pythonsdkpublish
+  orgIdentifier: default
+  projectIdentifier: IPA_Release
+  pipeline:
+    identifier: publish_python_sdk
+    properties:
+      ci:
+        codebase:
+          build:
+            type: tag
+            spec:
+              tag: <+trigger.tag>
+    stages:
+      - stage:
+          identifier: build_info
+          type: CI
+          spec:
+            infrastructure:
+              type: KubernetesDirect
+              spec:
+                nodeSelector:
+                  node_group: <+input>
```

Comment:
how do these inputs get populated? it looks like a node taint issue in harness is causing the build to fail

Reply:
https://github.com/IndicoDataSolutions/indico-client-python/blob/master-patch-1/.harness/publish_python_sdk.yaml#L30